packages/tools/dtc/package.mk

```diff
@@ -17,27 +17,38 @@
 ################################################################################
 
 PKG_NAME="dtc"
-PKG_VERSION="1.4.4"
-PKG_SHA256="2d1226634d71655466ebbd090d2873068c4918fbd5c433b91ead8ad08b9a5843"
+PKG_VERSION="1.4.5"
+PKG_SHA256="cfb9394690ebec1e4f942ee0c3b863b660eb0c4ef85bab19429f30c3469a3415"
 PKG_ARCH="any"
 PKG_LICENSE="GPL"
 PKG_SITE="https://git.kernel.org/pub/scm/utils/dtc/dtc.git/"
 PKG_URL="https://git.kernel.org/pub/scm/utils/dtc/dtc.git/snapshot/$PKG_VERSION.tar.gz"
 PKG_SOURCE_DIR="$PKG_VERSION"
+PKG_DEPENDS_HOST="Python2:host swig:host"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_SECTION="tools"
 PKG_SHORTDESC="The Device Tree Compiler"
 PKG_LONGDESC="The Device Tree Compiler"
 PKG_AUTORECONF="no"
 
-PKG_MAKE_OPTS_TARGET="dtc"
+PKG_MAKE_OPTS_TARGET="dtc libfdt"
 
 makeinstall_target() {
   mkdir -p $INSTALL/usr/bin
     cp -P $PKG_BUILD/dtc $INSTALL/usr/bin
 }
 
+PKG_MAKE_OPTS_HOST="dtc libfdt"
```

Comment:
Not really a big fan of this style of intermingling variable assignments and function declarations... I'd be happier if the above assignment is moved immediately above that for `PKG_MAKE_OPTS_TARGET`.

Reply:
better now?

Reply:
👍